### PR TITLE
[author] ifvisible - adds npm auto update configuration

### PR DESCRIPTION
**Pull request by serkanyersen**
Adds npm configuration

Thanks

### CODE REVIEW

File: ajax/libs/ifvisible/package.json [hunk 1]
```diff
@@ -4,6 +4,13 @@
   "version": "1.0.3",
   "description": "Crossbrowser & lightweight way to check if user is looking at the page or interacting with it. (wrapper around HTML5 visibility api)",
   "main": "ifvisible.js",
+  "npmName": "ifvisible.js",
+  "npmFileMap": [{
+    "basePath": "/src/",
+    "files": [
+      "*.js"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/ifvisible/package.json]: What about the map file?

Comment (serkanyersen) [inline ajax/libs/ifvisible/package.json]: Previous versions didn't have the map file so I didn't add it here.

**Comment by telkomsel13** [comment]
#1

File: ajax/libs/ifvisible/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/ifvisible/package.json]: But it has now, don't you add it?

**Comment by serkanyersen** [comment]
@PeterDaveHello Updated now to include all files under `src/` map file is for coffee file so it needs to be there as well.

Thanks

**Comment by PeterDaveHello** [comment]
@serkanyersen: we don't support coffee script on cdn, so should not include them.

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2

**Comment by maruilian11** [comment]
@serkanyersen any update? thanks!

**Comment by PeterDaveHello** [comment]
Auto-update setup in 120e783 #6564.
@maruilian11 @Piicksarn you should check what do we need to here first, instead of meaning less comment.

**Pull request closed without merging**